src/common/transformations/tests/utils/compress_quantize_weights.cpp

```diff
@@ -391,3 +391,164 @@ TEST_F(TransformationTestsF, NegativeCompressQuantizeWeightsNonConstantInput) {
     comparator.enable(FunctionsComparator::CmpValues::CONST_VALUES);
     comparator.enable(FunctionsComparator::CmpValues::ACCURACY);
 }
+
+using CompressWeightsWithFakeConvertParams = std::tuple<bool,          // zero_point_absent
+                                                        std::string>;  // float8 type
+
+class CompressWeightsNoZeroPoint : public TransformationTestsF,
+                                   public testing::WithParamInterface<CompressWeightsWithFakeConvertParams> {};
+
+TEST_P(CompressWeightsNoZeroPoint, FakeConvert) {
+    const auto& param = GetParam();
+    bool zero_point_absent = std::get<0>(param);
+    std::string destination_type = std::get<1>(param);
+
+    {
+        auto weights = op::v0::Constant::create(element::f32,
+                                                Shape{3, 1, 2, 2},
+                                                {-0.01448f,
+                                                 -0.02314f,
+                                                 -0.02244f,
+                                                 -0.00090f,
+                                                 0.024261f,
+                                                 0.031921f,
+                                                 0.034088f,
+                                                 -0.0497f,
+                                                 -0.0588f,
+                                                 -0.04541f,
+                                                 -0.01281f,
+                                                 0.009109f});
+        auto scale = op::v0::Constant::create(element::f32, Shape{3, 1, 1, 1}, {54.50976f});
+        std::shared_ptr<op::v13::FakeConvert> fake_convert;
+        if (zero_point_absent) {
+            fake_convert = std::make_shared<op::v13::FakeConvert>(weights, scale, destination_type);
+        } else {
+            auto shift = op::v0::Constant::create(element::f32, Shape{3, 1, 1, 1}, {0.0f});
+            fake_convert = std::make_shared<op::v13::FakeConvert>(weights, scale, shift, destination_type);
+        }
+        model = std::make_shared<Model>(fake_convert, ParameterVector{});
+
+        manager.register_pass<ov::pass::CompressQuantizeWeights>();
+    }
+
+    {
+        // TODO: change when it's allowed to create a fp8 constant from fp32 values
+        std::vector<uint8_t> weights_data = destination_type == "f8e4m3" ? std::vector<uint8_t>{0xb5,  // -0.8125
```

Comment:
Creating F8 Constant with F32 values should be already enabled, such mechanism is tested by the following tests:
https://github.com/openvinotoolkit/openvino/blob/be274ac0fae29627eb4d83f806e57a7071f7aae1/src/core/tests/eval.cpp#L3789-L3805

Also `ov::float8_e4m3` and `ov::float8_e5m2` can be used as a type of the std::vector directly like: 
`std::vector<ov::float8_e4m3>` or `std::vector<ov::float8_e5m2>`